### PR TITLE
NOOS-454/update boilerplate and python

### PR DESCRIPTION
**Pull request by filiperosa**
JIRA Reference: [NOOS-454](https://noosenergy.atlassian.net/browse/NOOS-454)

[NOOS-454]: https://noosenergy.atlassian.net/browse/NOOS-454?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Config documentation:
+// https://containers.dev/implementors/json_reference/
+{
```
Comment (pieeri) [inline .devcontainer/devcontainer.json:3]: 🐼 

Maybe worth adding the VS code settings if adding a Dev container configuration?

**Review by pieeri — CHANGES_REQUESTED** [review]
Maybe what has been missing in such a small library is a `CHANGELOG.md`!

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (filiperosa) [reply .devcontainer/devcontainer.json:3]: Done 👍

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@
 version: 2.1
 
 default:
-  install: &poetry-install
-    pkg-manager: poetry
-    install-args: ''
-  filter: &filter-tags-only
+  install: &poetry_install
+    pkg_manager: poetry
+    install_args: ''
```
[the remark this answers is not in the document]
Comment (filiperosa) [reply .circleci/config.yml]: Well spotted, done! 👍

**Review by pieeri — APPROVED** [review]
🚢 !

**Pull request merged**